README.md

```diff
@@ -89,7 +108,9 @@ adds tasks to your project build using a naming convention on a per source set b
 Similarly, the Solidity plugin will add a:
 
    * `compileSolidity` task for the project `main` source set.
-   * `compile<SourceSet>Solidity` for each remaining source set 
+   * `compile<SourceSet>Solidity` for each remaining source set.
+   * `resolveSolidity` for every source set.
```

Comment:
`resolveSolidity` is not for every source set, it is global for all. So better say something like:

* `resolveSolidity` task for all project Solidity sources.